database/seeds/H5PFillInTheBlankLibSubmitButtonSeeder.php

```diff
@@ -331,7 +331,7 @@ private function updatedSemantics() {
                 "type": "boolean",
                 "default": false,
                 "optional": true,
-                "description": "This options is compatible only with the standalone activities"
+                "description": "This option is only applies to a standalone activity. The Submit button is required for grade passback to an LMS."
```

Comment:
@yuvaraj-curriki have you removed the old data? because if not then this seeder will duplicate the data into DB

Reply:
@mohsin-tkxel 

This seeder will update the column in database. So i hope there will not be any duplicate data in DB as it is update operation


 public function run()
    {
        $h5pFillInTheBlankLibParams = ['name' => "H5P.Blanks", "major_version" => 1, "minor_version" => 12];
        $h5pFillInTheBlankLib = DB::table('h5p_libraries')->where($h5pFillInTheBlankLibParams)->first();
        if ($h5pFillInTheBlankLib) {
            DB::table('h5p_libraries')->where($h5pFillInTheBlankLibParams)->update(['semantics' => $this->updatedSemantics()]);
        }
    }